cppcache/src/statistics/StatisticDescriptor.hpp

```diff
@@ -83,7 +83,7 @@ class APACHE_GEODE_EXPORT StatisticDescriptor {
   /*
```

Comment:
Can you please also cleanup useless comments around the areas you are changing. There are lots of "destructor" and "constructor" comments that are pretty darn obvious given the language.